CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ The changes are relative to the previous release, unless the baseline is specifi
 ### Removed since 1.3.0
 
 * Remove ext/avm.cmd.
+* Remove the AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM CMake flag.
```

Comment:
Should we add in the "Added since 1.3.0" section that sample transforms of AVIF v1.2 are offically supported? Then we may not need this item in the "Removed since 1.3.0" section.

Reply:
I added an entry and kept the other one. I think it is clearer.